support/reattach-pv/main.go

```diff
@@ -152,8 +162,8 @@ func checkClaimsNotFound(c k8s.Client, claims map[types.NamespacedName]v1.Persis
 }
 
 // expectedVolumeClaims builds the list of PersistentVolumeClaim that we expect to exist for the given
-// Elasticsearch cluster.
-func expectedVolumeClaims(es esv1.Elasticsearch) map[types.NamespacedName]v1.PersistentVolumeClaim {
+// Elasticsearch cluster, or oldEsName, if given.
+func expectedVolumeClaims(es esv1.Elasticsearch, oldEsName string) map[types.NamespacedName]v1.PersistentVolumeClaim {
```

Comment:
`oldEsName` is not used in this function

Reply:
yeah it was used previously.  Removed, thanks.